### ISSUE TITLE
Icon Captcha

### ISSUE THREAD

**Issue by hakanyakar** [cycyup/crack_geetest#4]
Does this work for geetest icon captcha?

**Comment by cycyup** [comment]
What is icon captcha?

**Comment by hakanyakar** [comment]
![image](https://user-images.githubusercontent.com/21169034/158011858-7d47b706-1bc8-4f2e-a8a0-8b7e2dcc8799.png)
this is icon captcha how can i bypass this kind of captcha on backend

**Comment by cycyup** [comment]
This only can bypass space captcha

**Comment by hakanyakar** [comment]
humm got it thanks. My another question is what is the w parameter I send in payload do you know and how can i generate it?
Is it releated with lot_number?
![image](https://user-images.githubusercontent.com/21169034/158012272-0240971f-afb4-4462-960d-8f00f8c00f2f.png)

**Comment by cycyup** [comment]
If you are good at Chinese, you can read https://mp.csdn.net/mp_blog/creation/editor/122953184. I explained how to generate w parameter

**Comment by cycyup** [comment]
You can deobfuscator js and debug how to generate w paramter

**Comment by hakanyakar** [comment]
Thank you very much but I I can not register to website to read your blog

**Comment by hakanyakar** [comment]
I couldn't check the link you shared. It requires sign in. Then I searched in this blog for solutuion. I found this blog
( https://blog.csdn.net/web15185420056/article/details/123082123?ops_request_misc=%257B%2522request%255Fid%2522%253A%2522164708181316780264029920%2522%252C%2522scm%2522%253A%252220140713.130102334.pc%255Fall.%2522%257D&request_id=164708181316780264029920&biz_id=0&utm_medium=distribute.pc_search_result.none-task-blog-2~all~first_rank_ecpm_v1~rank_v31_ecpm-2-123082123.first_rank_v2_pc_rank_v29_v2&utm_term=geetest+v4#2_w_148 )

I did step by step what he said but still I couldn't generate "w" parameter for icon captcha.
Is there any way to bypass geetest captcha without generating "w" and without image recognition.

**Comment by cycyup** [comment]
no, can't bypass. but why couldn't generate "w" parameter for icon captcha?

**Comment by hakanyakar** [comment]
It is so complex. So much encrypted things. I can't figure it out how to extract "w" parameter. I try to debug in chrome console but I lost inside the code :)))

**Comment by loayei** [comment]
@hakanyakar Were you able to bypass the captcha you mentioned?

**Comment by hakanyakar** [comment]
@loayei  No, I found an another solution. But new solutions always come with new problems. Maybe You can help me out this. Can I send you an email for this?

**Comment by loayei** [comment]
You can contact me on discord better Loayei#4671

**Comment by Ljzd-PRO** [comment]
> @loayei No, I found an another solution. But new solutions always come with new problems. Maybe You can help me out this. Can I send you an email for this?

What solution?
